dlf/plugins/pageview/tx_dlf_utils.js

```diff
@@ -561,38 +561,48 @@ dlfUtils.isCorsEnabled = function(imageObjs) {
     // fetch data from server
     // with access control allowed
     let response = true;
-
+    const type = 'HEAD';
     imageObjs.forEach(function(imageObj) {
         let url = imageObj.mimetype === dlfUtils.CUSTOM_MIMETYPE.ZOOMIFY
-            ? imageObj.url.replace('ImageProperties.xml', 'TileGroup0/0-0-0.jpg')
-            :
-            imageObj.mimetype === dlfUtils.CUSTOM_MIMETYPE.IIIF
-                ? dlfViewerSource.IIIF.getMetdadataURL(imageObj.url)
+            ? dlfUtils.buildXhrURI(imageObj.url.replace('ImageProperties.xml', 'TileGroup0/0-0-0.jpg'), true)
+            : imageObj.mimetype === dlfUtils.CUSTOM_MIMETYPE.IIIF
+                ? dlfUtils.buildXhrURI(dlfViewerSource.IIIF.getMetdadataURL(imageObj.url), false)
                 : imageObj.mimetype === dlfUtils.CUSTOM_MIMETYPE.IIP
-                ? dlfViewerSource.IIP.getMetdadataURL(imageObj.url)
-                : imageObj.url;
-
-        url =
-            window.location.origin + window.location.pathname + '?url=' + encodeURIComponent(
-                url) + '&header=2';
+                    ? dlfUtils.buildXhrURI(dlfViewerSource.IIP.getMetdadataURL(imageObj.url), true)
+                    : dlfUtils.buildXhrURI(imageObj.url, true);
 
         $.ajax({
             url: url,
+            type: type,
```

Comment:
Good idea. But not sure, if this change a lot. With "&header=2" the eID does anyway only a HEAD request by the server. At least, it should do ;-) I have to examine this later.